### DISCUSSION TITLE
Any easy way to import a style file on every page

### DISCUSSION THREAD

**Discussion by CrystalWindSnake** [zauberzeug/nicegui#1631] [Q&A] [answered]
### Question

I have a custom component with a style file. I hope that when it is used, the caller does not need to explicitly import the style file.

cus component file `fontColor.py`
```python
from nicegui import ui
from pathlib import Path

css_file = Path(__file__).parent / "fontColor.css"
ui.add_head_html(f"""<style>{css_file.read_text()}</style>""")

class FontColor(ui.element):
    def __init__(
        self,
    ) -> None:
        super().__init__("div")
        self.props("data-v-09ce85c6")

```

`fontColor.css`
```css
div[data-v-09ce85c6]>* {
    background-color: red;
}
```

`main.py`
```python
from nicegui import ui
from fontColor import FontColor

with FontColor():
    ui.label("bg color should be red")


@ui.page("/other")
def index():
    # this page not work
    with FontColor():
        ui.label("bg color should be red")


ui.run()
```

localhost page displayed normally.but the /other page not work.

In the custom component, there is a parameter `extra_libraries` to set dependent JS files. But there is no item to set dependent CSS files.

how can I do that? thanks.

**Comment by rodja** [comment, marked as answer]
You need to call `ui.add_head_html` in every page.

**Reply by CrystalWindSnake** [reply]
got it, thank you for your quick response.
